src/Umbraco.Web/Models/Mapping/EntityModelMapper.cs

```diff
@@ -17,18 +18,21 @@ internal class EntityModelMapper : MapperConfiguration
         public override void ConfigureMappings(IConfiguration config, ApplicationContext applicationContext)
         {
             config.CreateMap<UmbracoEntity, EntityBasic>()
+                .ForMember(x => x.Udi, expression => expression.MapFrom(x => Udi.Create(UmbracoObjectTypesExtensions.GetUdiType(x.NodeObjectTypeId), x.Key))) 
                 .ForMember(basic => basic.Icon, expression => expression.MapFrom(entity => entity.ContentTypeIcon))
                 .ForMember(dto => dto.Trashed, expression => expression.Ignore())
                 .ForMember(x => x.Alias, expression => expression.Ignore());
 
             config.CreateMap<PropertyType, EntityBasic>()
+                .ForMember(x => x.Udi, expression => expression.Ignore())

```

Comment:
Correct and if you don't ignore this than the auto mapper tests will fail saying you are missing a mapping.